documents/motions/16.2021.1 - Disputes.md

```diff
@@ -1,23 +1,36 @@
-1. All acts and decisions of WCA Staff may be subject to disputes by affected individuals.
-   1. Disputes regarding the WCA Regulations must be directed to and handled by WCA Regulations Committee, and are bound by the provisions of the WCA Regulations.
-      1. Appeals regarding the final decision must be directed to and handled by the WCA Board.
-   2. Disputes regarding the WCA Bylaws and the Motions should be directed to and handled by the WCA Board, and are bound by the provisions of the WCA Bylaws and the Motions and the applicable governing laws.
-      1. Appeals regarding the final decision should be directed to and handled by WCA Ethics Committee.
-   3. All other Disputes should be directed to and handled by WCA Disciplinary Committee, and are bound by the WCA Regulations, the WCA Bylaws and the Motions, and the applicable governing laws.
-      1. Appeals regarding the final decision should be directed to and handled by WCA Ethics Committee.
-2. Disputes must be filed within sixty days after the disputed act or decision.
-3. Disputes must be handled within sixty days of the date of dispute.
-4. Appeals to decisions on disputes must be filed within sixty days after the decision.
-5. Appeals must be handled within sixty days of the date of appeal.
-6. During the process of disputes and appeals all involved parties should be heard.
-7. The decisions on disputes and appeals will take immediate effect and all WCA Staff, Registered Speedcubers, and Partners must take all necessary actions to ensure that the decisions come into effect.
+1. Acts and decisions of WCA Staff may be subject to disputes by affected individuals or organizations.
+   1. Disputes regarding the application of WCA Regulations must be directed to and handled by WCA Regulations Committee, and are bound by the provisions of the WCA Regulations.
+   2. Disputes regarding the conduct of WCA Staff where it is alleged the Spirit, Mission, Motions, Policies, Codes, Regulations and the applicable governing laws were violated must be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws.
+   3. Disputes regarding the conduct of WCA Community members where it is alleged the Spirit, Mission, Motions, Policies, Codes, Regulations and the applicable governing laws were violated must be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws. 
+   4. All other Disputes should be directed to and handled by WCA Integrity Committee, and are bound by the Spirit, Mission, Motions, Policies, Codes, Regulations of the WCA, and the applicable governing laws.
+2. The decision made by the Committee handling the dispute may be subject to appeal by affected individuals or organizations.
+   1. Appeals on disputes shall be handled by the WCA Appeals Committee.
```

Comment:
Any reason to use "shall" instead of "must" (which is usually the preferred option for WCA Documents)?

Considering RFC 2119, I assume both have the same meaning, but it could be ambiguous for anyone used to "may", "should" and "must".